### DISCUSSION TITLE
PnP Search error: [HTTP]:500 - [CorrelationId]:15a533a1-904e-5000-ac0b-857d94435f09 [Version]:16.0.0.24929

### DISCUSSION THREAD

**Discussion by afnepipe** [microsoft-search/pnp-modern-search#3807] [Q&A]
Hello everyone, @wobba 
I appreciate who can help me with this error.
I saw that there are a couple of similar discussions, but they don't exactly fit my case.

**Error image:**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/8c63234a-f270-4805-bfda-c60b6e6f8f3e)

**Current configuration:**
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/8b497762-bcc2-4e2b-bebf-f9e5e7fc0f89)

PnP - Search Results
Version: 4.8.0

I appreciate any recommendation.

**Comment by wobba** [comment]
A 500 error means something went wrong. So looking at the search request for any errors may help.

**Reply by afnepipe** [reply]
Thanks @wobba
I just updated the application from 4.8 to 4.11.1 in the hope that it might work.
The search sometimes fails and other times it works fine.

I have set up a test environment with the new version. In this environment, the queries work fine. This environment has about 40 documents. While production has more than 7000.

I was thinking, the error could be in the Result Source, but is working well.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/96144314/211b27f5-d806-4091-be36-a6ed50ca020b)


Do you have any other suggestions?

**Reply by wobba** [reply]
If it's intermittent for the exact same query, then it's an API issue, not a web part issue.
